go.sum

```diff
@@ -241,6 +241,7 @@ github.com/kr/pretty v0.1.0/go.mod h1:dAy3ld7l9f0ibDNOQOHHMYYIIbhfbHSm3C4ZsoJORN
 github.com/kr/pty v1.1.1/go.mod h1:pFQYn66WHrOpPYNljwOMqo10TkYh1fy3cYio2l3bCsQ=
 github.com/kr/text v0.1.0 h1:45sCR5RtlFHMR4UwH9sdQ5TC8v0qDQCHnXt+kaKSTVE=
 github.com/kr/text v0.1.0/go.mod h1:4Jbv+DJW3UT/LiOwJeYQe1efqtUx/iVham/4vfdArNI=
+github.com/kr/text v0.2.0 h1:5Nx0Ya0ZqY2ygV366QzturHI13Jq95ApcVaJBhpS+AY=
```

Comment:
Are you sure this is to be commited? I see no changes in deps

Reply:
There was some changes in go.sum when I was building the image.

Reply:
Can you please just check it again I tried on my local & go mod tidy removes reverts back to original?

Reply:
Cool it works!

Reply:
merge! :)

Reply:
I was talking about go-deps 😆 , it's a pain to resolve go mod conflicts tbh. @kalradev is still building the local docker image to verify the changes 🚀